src/platform-includes/performance/enable-tracing/react-native.mdx

```diff
@@ -1,10 +1,8 @@
-<Alert level="Info" title="Prior Versions">
+Tracing is enabled by default in the React Native SDK.
 
-If you use a version of our SDK prior to version `3.0.0`, you will need to include the `ReactNativeTracing` integration to use automatic instrumentation. You do not need to do this in versions `3.0.0` and above.
-
-</Alert>
+<Expandable title="Using older Sentry for React Native? Here is how to enabled Tracing">
 
-<SignInNote />
+If you use a version of our SDK prior to version `3.0.0`, you will need to include the `ReactNativeTracing` integration to use automatic instrumentation. You do not need to do this in versions `3.0.0` and above.
```

Comment:
We can make `ReactNativeTracing` also a link to the docs that tells you how to set it up